packages/shortest/src/types/browser.ts

```diff
@@ -43,6 +45,8 @@ export interface ActionInput {
   username?: string;
   password?: string;
   url?: string;
+  duration?: number;
+  email?: string;
```

Comment:
these are action input types. Helps us to validate that AI is sending the required inputs.